.github/workflows/deploy_to_environment.yml

```diff
@@ -64,45 +59,53 @@ jobs:
           key: v1-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
             v1-yarn-
-      - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@67fbcbb121271f7775d2e7715933280b06314838
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: us-west-2
           role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
-          role-duration-seconds: 1800
-          role-skip-session-tagging: true
+          aws-region: us-east-1
       - name: Clean the database
         if: contains(inputs.env, 'dev')
+        env:
+          ECR_REGISTRY: ${{ secrets.ECR_REGISTRY }}
+          ECR_REPOSITORY: mint-db-clean
+          NEW_IMAGE_TAG: latest
+          TASK_FAMILY: "db-clean-${{ inputs.env }}"
+          ECS_CLUSTER: "${{ inputs.env }}-mint-app"
+          SECURITY_GROUP_NAME: "ecs-db-clean-${{ inputs.env }}"
         run: |
-          ./scripts/db_lambda_invoke "easi-app-db-clean-tmp" "2" "easi-app-db-clean"
-      - name: Backup the database
-        if: contains(inputs.env, 'impl') || contains(inputs.env, 'prod')
-        run: |
-          ./scripts/rds-snapshot-app-db
-      - name: Run migrations
+          export TASK_REVISION=$(./scripts/update_ecs_task_definition.sh)
+          ./scripts/run_ecs_task.sh
+      - name: Run database migrations
         env:
-          AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
           ECR_REGISTRY: ${{ secrets.ECR_REGISTRY }}
-          ECR_REPOSITORY: easi-db-migrate
-          LAMBDA_VERSION: ${{ inputs.lambda_version }}
-          ENV: ${{ inputs.env }}
+          ECR_REPOSITORY: mint-db-migrate
+          NEW_IMAGE_TAG: ${{ env.GIT_HASH }}
+          TASK_FAMILY: "db-migrate-${{ inputs.env }}"
+          ECS_CLUSTER: "${{ inputs.env }}-mint-app"
+          SECURITY_GROUP_NAME: "ecs-db-migrate-${{ inputs.env }}"
         run: |
-          ./scripts/deploy_service "easi-app-db-migrate" "$ENV-ecs-manager-docker" "$LAMBDA_VERSION" "easi-db-migrate" "$ECR_REGISTRY/$ECR_REPOSITORY:$GIT_HASH"
-          ./scripts/db_lambda_invoke "$ENV-ecs-manager-docker" "$LAMBDA_VERSION" "easi-app-db-migrate"
+          export TASK_REVISION=$(./scripts/update_ecs_task_definition.sh)
+          ./scripts/run_ecs_task.sh
+      # - name: Backup the database
+      #   if: contains(inputs.env, 'impl') || contains(inputs.env, 'prod')
+      #   run: |
+      #     ./scripts/rds-snapshot-app-db
       - name: Deploy ECS service
```

Comment:
Maybe in a future iteration we could look into using this [amazon-ecs-deploy-task-definition](https://github.com/aws-actions/amazon-ecs-deploy-task-definition) action